core/conn/jdbcT4/Makefile

```diff
@@ -26,17 +26,20 @@ include ../../macros.gmk #top level
 all: build_all
 
 build_all: LICENSE NOTICE 
-	echo "$(MAVEN) package -DskipTests"
-	set -o pipefail && $(MAVEN) package -DskipTests | tee build_jdbct4.log | grep --line-buffered -E -e '^\[[^WId]' -e '^\[INFO\] B[Uu][Ii][Ll][Dd]' -e 'to compile'
-	cp target/jdbcT4-${TRAFODION_VER}.jar ${TRAF_HOME}/export/lib
-	mkdir -p ../clients
-	mv target/jdbcT4-${TRAFODION_VER}.zip ../clients
-	#ln -sf ${TRAF_HOME}/export/lib/jdbcT4-${TRAFODION_VER}.jar ${TRAF_HOME}/export/lib/jdbcT4.jar
-	`cd ${TRAF_HOME}/export/lib;ln -sf jdbcT4-${TRAFODION_VER}.jar jdbcT4.jar`
+	@if [ ! -f target/*.jar ]; then \
```

Comment:
Instead of target/*.jar, should it check for the exact file that we are trying to build. What if TRAFODION_VER changes or there is some other JAR already in the 'target' directory.

Reply:
If *.jar expands to more than one file, won't that generate a shell syntax error?

Reply:
@narendragoyal @svarnau  sure, it should be an exactly name.